### PR TITLE
Add Perf troubleshooting tips

### PR DESCRIPTION
**Pull request by RenaShahMSFT**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: docs/csi-debug.md [hunk 1]
```diff
@@ -53,3 +53,18 @@ dir
...
+
+
+### Troubleshooting performance issues on Azure Files
+
+##### File shares are being throttled and overall performance is slow 
+Find out whether you have been throttled by looking at your Azure Monitor by following this documentation - [Link](https://docs.microsoft.com/en-us/azure/storage/files/storage-troubleshooting-files-performance#cause-1-share-was-throttled)
+
+###### Standard Files
+
+Enable [larger file shares](https://docs.microsoft.com/azure/storage/files/storage-files-how-to-create-large-file-share?tabs=azure-portal) on your storage account. Large file shares support up to 10,000 IOPS per share at no extra cost on non-premium tier. Right now, Large File Shares need to be manually turned on the storage account and quota needs to be set to 100 TiB to get 10K IOPS. In general - one does not need remounting of the file share after the setting is changed, however it is recommended to turn it on at Storage Account creation time. Same storage account can have multiple LargeFileShare in Standard Storage. We are looking into providing it as an option on the CSI driver in future.
```
Comment (VybavaRamadoss) [inline docs/csi-debug.md:65]: Suggest saying Standard to stick to names we use today and pricing page instead of non-premium.

Comment (VybavaRamadoss) [inline docs/csi-debug.md:65]: Rephrase "In general - one does not need remounting of the file share after the setting is changed, however it is recommended to turn it on at Storage Account creation time." to "We recommend that you enable this when creating the Storage Account. However, if you do enable this at a later time, you will not need to remount the file share. "

Comment (VybavaRamadoss) [inline docs/csi-debug.md:65]: Large File Share vs LargeFileShare

Comment (VybavaRamadoss) [inline docs/csi-debug.md:65]: Recommend removing this : We are looking into providing it as an option on the CSI driver in future.
I think we should do this anyway. You can also see how many people add this manually on the file shares.

Comment (VybavaRamadoss) [inline docs/csi-debug.md:65]: Suggest saying Standard to stick to names we use today and pricing page instead of non-premium.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**